utilities/plotting/useLivePlot.m

```diff
@@ -0,0 +1,45 @@
+classdef useLivePlot < handle
+    % Sets up a live plot on a given figure 
+    properties
+        figureId
+        handle
+    end
+
+    properties (Constant)
+        callback = @plotRefSLDHelper
+    end
+
+    methods
+        function obj = useLivePlot(figureId)
+            arguments
+                figureId {mustBePositive, mustBeInteger} = 1000
+            end
+            obj.figureId = figureId;
+            obj.handle = @(varargin) obj.callback(varargin{:}, true, obj.figureId);
+            % Make the figure
+            h = figure(obj.figureId);
+            subplot(1,2,1);
+            subplot(1,2,2);
```

Comment:
Hmm, are we hard-wiring here?

Reply:
Ok, this is kind of already the case in "triggerEvent", but maybe something to think about.

Reply:
I will remove the subplot calls